lms/djangoapps/instructor/enrollment.py

```diff
@@ -267,6 +269,15 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
 
     if delete_module:
         module_to_reset.delete()
+        if block and block.has_score:
+            SCORE_CHANGED.send(
+                sender=None,
+                points_possible=block.max_score,
```

Comment:
Actually, I don't believe this is the correct value for `points_possible`.  The block's weight also needs to be taken into consideration.  Is it necessary to give a value for `points_possible` when the score is reset?